src/app/shared/dialog/facet-entries/facet-entries.component.html

```diff
@@ -15,13 +15,13 @@
             <td scope="row" [ngSwitch]="facet.type">
               <span *ngSwitchCase="'STRING'">
                 <a [routerLink]="routerLink" [queryParams]="getQueryParams(queryParams | async, facet, entry)" queryParamsHandling="merge">
-                  <span [innerHTML]="entry.value | formalize | safeHtml"></span>
+                  <span [innerHTML]="getStringValue(entry) | formalize | safeHtml"></span>
```

Comment:
Must this be explicitly converted into a string via a function call instead of using something like a filter `entry | getStringValue` ?

Reply:
I could use a filter, but this was just easier. Not sure if one is recommended over the other. The method name could be better named. It is to return valueHtml if exists otherwise, return value.

`entry.valueHtml ? entry.valueHtml : entry.value`

Reply:
In order to use the pipe, `getStringValue` would have to be a pipe and I don't think this justifies creating one.